src/styles/utils/_mixins.scss

```diff
@@ -0,0 +1,17 @@
+@mixin onDesktop {
+  @media (min-width: 1200px) {
+    @content;
+  }
+}
+
+@mixin onTablet {
+  @media (min-width: 640px) and (max-width: 1199px) {
+    @content;
+  }
+}
+
+@mixin onPhone {
+  @media (max-width: 639px) {
+    @content;
+  }
+}
```

Comment:
As I said, you don't need this. Because your basic markup is written for phone screens (in grid.scss you don't use mixin onPhone).And also you can remove max-width onTablet.

```suggestion

```